tests/types/hexColorCode.spec.ts

```diff
@@ -0,0 +1,74 @@
+import {StrongHexCode, makeHexCode} from '../../src/types/hexColorCode';
+
+const MOCK_INITIAL = '#ffffff';
+const MOCK_FALLBACK_DEFAULT = 'FF5733';
+const MOCK_FALLBACK = '33B5FF';
+
+describe('StrongHexCode', () => {
```

Comment:
Updating test's root describe block label to match updated name of StrongHexColorCode.

- [ ] Change `StrongHexCode` describe label to `StrongHexColorCode`.